SarandONGa/urls.py

```diff
@@ -33,5 +34,8 @@
     path('payment/list', payment_views.payment_list, name="payment_list"),
     path('donations/', include(donation_urls), name='donations'),
     path('subsidy/list', subsidy_views.subsidy_list, name="subsidy"),
+    path('service/', include(service_urls),name="service"),
```

Comment:
From what I've seen many teammates are indicating the creation path with /create, I think you could change it to **service/create**

Reply:
oke